doc/release/0.18.0-notes.rst

```diff
@@ -62,6 +62,10 @@ is now consistently added after the matrix is applied,
 independent of if the matrix is specified using a one-dimensional
 or a two-dimensional array.
 
+The `mode` keyword argument in ``signal.spectrogram`` no longer supports the
```

Comment:
Right on, thanks. I haven't done the legwork to try and render the docs locally...